monkey/monkey_island/cc/ui/src/components/utils/ServerUtils.tsx

```diff
@@ -125,3 +125,14 @@ export function getMachineByAgent(agent, machines) {
 
   return agentMachine;
 }
+
+export function getIslandIPsFromMachines(machines) {
+  let ips = [];
+  for(let machine of machines) {
+    if(machine.island){
+      ips = getMachineIPs(machine);
+    }
+    break;
```

Comment:
Wouldn't this kick us out of the for-loop after the first iteration? Why not just return in the if-statement?

Reply:
+1